docs/src/main/asciidoc/doc-reference.adoc

```diff
@@ -303,11 +316,39 @@ xref:<other-file-name>.adoc#title-heading[Title heading]
 ----
 xref:<name-of-the-file>.adoc[Human-readable label]
 ----
-For more details about anchored IDs, see the xref:doc-contribute-docs-howto.adoc#anchors-howto[Cross-reference in-file and cross-file content by using anchors] section.
+For more details about anchored IDs, see the xref:doc-contribute-docs-howto.adoc#anchors-howto[Cross-reference in-file and cross-file content by using anchors] section of the "Contribute to Quarkus documentation" guide.
+
+=== Cross-reference phrasing
+
+For consistency and context clarity, use the following guidelines for constructing your cross-reference phrases:
 
-=== Reference source code 
+* Provide a direct hyperlink to the section, using the `xref` guidance in the previous section.
+* Specify the full title of the section and use sentence case.
+* Add the definite article “the” before the section title and specify  “section” directly after the title.
+* Add the definite article “the” and “Quarkus” before the title of the guide, unlessthe word “Quarkus" is already in the title.
```

Comment:
```suggestion
* Add the definite article “the” and “Quarkus” before the title of the guide, unless the word “Quarkus" is already in the title.
```

Reply:
@sheilamjones - Good catch! Thank you. I fixed this in my last commit with a batch of other changes. It's now appearing OK on line 344.